src/app/app.module.ts

```diff
@@ -100,6 +119,21 @@ import { CardComponent } from './card/card.component';
     HttpClientModule,
     FormsModule,
     BrowserAnimationsModule,
+    AgmCoreModule.forRoot({
+      apiKey: 'AIzaSyCKNBwrs1UdT2s1jwqOypSzas9Z4s6h4B0'
```

Comment:
1) Never commit API keys
2) I thought we were not going to use google maps here?